.github/workflows/jenkins-x-pr.yaml

```diff
@@ -14,29 +17,41 @@ jobs:
     - env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       name: build-make-linux
-      uses: docker://golang:1.17.9
+      uses: docker://golang:1.19.3
       with:
         args: -c "make linux"
         entrypoint: /bin/sh
     - env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       name: build-make-test
-      uses: docker://golang:1.17.9
+      uses: docker://golang:1.19.3
       with:
         args: -c "make test"
         entrypoint: /bin/sh
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v1
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GHCR_TOKEN }}
     - name: Build and push
       uses: docker/build-push-action@v2
       with:
         context: .
         file: ./Dockerfile
-        push: false
-"on":
-  push:
-    branches-ignore:
+        platforms: linux/amd64,linux/arm64
+        push: true
+        build-args: |
```

Comment:
can you set tags to pr-debug? No version is set, and latest should come from release branch